frontend/src/components/modals/JoinLinkNoAccessModal.tsx

```diff
@@ -0,0 +1,70 @@
+import { FormEvent } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Form } from 'react-bootstrap';
+import { Trans } from 'react-i18next';
+import { Modal, ModalTypes } from './types';
+import { ModalContent } from './modalparts/ModalContent';
+import { ModalFooter } from './modalparts/ModalFooter';
+import { ModalFooterButtonDiv } from './modalparts/ModalFooterButtonDiv';
+import { ModalHeader } from './modalparts/ModalHeader';
+import { paths } from '../../routers/paths';
+import { ReactComponent as AlertOrangeIcon } from '../../icons/alert_orange_icon.svg';
+import '../../shared.scss';
+
+export const JoinLinkNoAccessModal: Modal<ModalTypes.JOIN_LINK_NO_ACCESS_MODAL> = ({
+  closeModal,
+  invitationLink,
+}) => {
+  const history = useHistory();
+
+  const closeAndRedirectToOverview = () => {
+    closeModal();
+    history.push(`${paths.overview}`);
+  };
```

Comment:
This function is used in all of the invitation modals. I wanted to provide it in the props (like closeModal) but could not figure out how